README.md

```diff
@@ -1,2 +1,4 @@
 # git-demo
 to demonstrate how to use git and GitHub
+
+-[Jacqueline Chen](https://github.com/CCccc-76)
```

Comment:
Could you add a space between the `-` and your name? Looks great otherwise!

Reply:
Just revised it!